undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd

```diff
@@ -199,6 +199,7 @@
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="alias" use="optional" type="xs:string"/>
         <xs:attribute name="default-web-module" use="optional" type="xs:string" default="ROOT.war"/>
+        <xs:attribute name="default-response-code" use="optional" type="xs:int"/>
```

Comment:
This could use some xs:documentation. It's not obvious under what conditions someone would set this.